scripts/trial/TrialCustomSettingsDefaults.cls

```diff
@@ -22,26 +22,27 @@ public static void setContactsAndOrgDefaults() {
     settingsToUpdate.add(contactSettings);
 }
 
-public static void setRelationshipDefaults() {
+private static void setRelationshipDefaults() {
     npe4__Relationship_Settings__c relationshipSettings = npe4__Relationship_Settings__c.getOrgDefaults();
     relationshipSettings.npe4__Gender_Field__c = 'Gender__c';
 
     settingsToUpdate.add(relationshipSettings);
 }
 
-public static void setRecurringDonationDefaults() {
+private static void setRecurringDonationDefaults() {
     npe03__Recurring_Donations_Settings__c recurringDonationsSettings =
         npe03__Recurring_Donations_Settings__c.getOrgDefaults();
 
     recurringDonationsSettings.%%%NAMESPACE%%%RecurringDonationNameFormat__c =
             '{!npe03__Donor_Name__c} {!npe03__Amount__c} - Recurring';
     recurringDonationsSettings.%%%NAMESPACE%%%EnableAutomaticNaming__c = true;
     recurringDonationsSettings.npe03__Record_Type__c = getOppRecordType('Donation');
-
+    recurringDonationsSettings.%%%NAMESPACE%%%StatusAutomationLapsedValue__c = 'Lapsed';
+    recurringDonationsSettings.%%%NAMESPACE%%%StatusAutomationClosedValue__c = 'Closed';
```

Comment:
@andrewyu-salesforce Yeah, we don't intend to automatically enable the feature since we are not sure what standard values a customer might set. Populating the picklist fields is just so those values can be defaulted since there's only one option available anyway.